libs/feature/record/src/lib/state/mdview.selectors.ts

```diff
@@ -64,3 +64,11 @@ export const getAddUserFeedbacksLoading = createSelector(
   getMdViewState,
   (state: MetadataViewState) => state.addUserFeedbackLoading
 )
+
+/*
```

Comment:
You can format as a jsdoc like this:
```suggestion
/**
 * Feature Catalog Selectors
 * @param xxx ....
 */
```

If you just type /** and enter VSCode will autocomplete with a js doc block. Maybe if you really want to describe the function you can also add a sentence explaining what it does excatly.

Reply:
Oh sorry I did not realized you were commenting sections and not functions. Ignore my comment 😅...